### PR TITLE
DEV: flesh out FT.INFO page

### PR DESCRIPTION
**Pull request by dwdougherty**
Field definitions are [here](https://redislabs.atlassian.net/wiki/spaces/DX/pages/3832414265/FT.INFO+Interpretation+Guide).

@joeywhelan Would it be possible for you to provide a definition for `cleaning`, one of the return values? Right now, I just have a placeholder where its definition should be, "DWD need def'n".

### CODE REVIEW

File: content/commands/ft.info/index.md [hunk 1]
```diff
@@ -29,140 +29,326 @@ syntax_str: ''
...
 
-## Return
+### Indexing-related statistics
 
-FT.INFO returns an array reply with pairs of keys and values.
+| Statistic | Definition |
+|:---       |:---        |
+| `hash_indexing_failures` | The number of failures encountered during indexing. |
+| `total_indexing_time` | The total time is taken for indexing in seconds. |
+| `indexing` | Indicates whether the index is currently being indexed. |
```
Comment (andy-stark-redis) [inline content/commands/ft.info/index.md:82]: Looks a bit odd like that. Maybe "Indicates whether the data is currently being indexed" is another option.
```suggestion
| `indexing` | Indicates whether the index is currently being generated. |
```

File: content/commands/ft.info/index.md [hunk 2]
```diff
@@ -29,140 +29,326 @@ syntax_str: ''
...
+| `percent_indexed` | The percentage of the index that has been successfully indexed. |
```
Comment (andy-stark-redis) [inline content/commands/ft.info/index.md:83]:
```suggestion
| `percent_indexed` | The percentage of the index that has been successfully generated. |
```

File: content/commands/ft.info/index.md [hunk 3]
```diff
@@ -29,140 +29,326 @@ syntax_str: ''
...
+| Statistic | Definition |
+|:---       |:---        |
+| `bytes_collected` | The number of bytes collected during garbage collection. |
+| `total_ms_run` | The total time in milliseconds spent on garbage collection. |
+| `total_cycles` | The total number of garbage collection cycles. |
+| `average_cycle_time_ms` | The average time in milliseconds for each garbage collection cycle. The value `nan` indicates that the average cycle time is not available. |
+| `last_run_time_ms` | The time in milliseconds taken by the last garbage collection run. |
 
-Optional statistics include:
+The next two GC-related fields are relevant in scenarios where simultaneous changes occurred in the same memory area by both the parent process and the child process, resulting in the parent discarding these changes.
```
Comment (andy-stark-redis) [inline content/commands/ft.info/index.md:97]:
```suggestion
The next two GC-related fields are relevant in scenarios where simultaneous changes occurred in the same memory area for both the parent process and the child process, resulting in the parent discarding these changes.
```

File: content/commands/ft.info/index.md [hunk 4]
```diff
@@ -29,140 +29,326 @@ syntax_str: ''
...
 
-FT.INFO returns an array reply with pairs of keys and values.
+| Statistic | Definition |
+|:---       |:---        |
+| `hash_indexing_failures` | The number of failures encountered during indexing. |
+| `total_indexing_time` | The total time taken for indexing in seconds. |
+| `indexing` | Indicates whether the index is currently being generated. |
+| `percent_indexed` | The percentage of the index that has been successfully generated. |
+| `number_of_uses` | The number of times the index has been used. |
+| `cleaning` | DWD need def'n |
```
Comment (dwdougherty) [inline content/commands/ft.info/index.md:85]:
```suggestion
| `cleaning` | The index deletion flag. A value of `1` is an indication that index deletion is in progress. |
```

File: content/commands/ft.info/index.md [hunk 5]
```diff
@@ -29,140 +29,326 @@ syntax_str: ''
...
 
-FT.INFO returns an array reply with pairs of keys and values.
+| Statistic | Definition |
+|:---       |:---        |
+| `hash_indexing_failures` | The number of failures encountered during indexing. |
+| `total_indexing_time` | The total time taken for indexing in seconds. |
+| `indexing` | Indicates whether the index is currently being generated. |
+| `percent_indexed` | The percentage of the index that has been successfully generated. |
+| `number_of_uses` | The number of times the index has been used. |
+| `cleaning` | The index deletion flag. A value of `1` is an indication that index deletion is in progress. |
```
Comment (dwdougherty) [inline content/commands/ft.info/index.md:85]:
```suggestion
| `cleaning` | The index deletion flag. A value of `1` indicates index deletion is in progress. |
```

**Pull request merged**